torch/csrc/api/include/torch/nn/functional/batchnorm.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+#include <torch/types.h>
+#include <torch/cuda.h>
+
+namespace torch {
+namespace nn {
+namespace functional {
+
+inline Tensor batch_norm(const Tensor& input, const Tensor& running_mean,
+                         const Tensor& running_var, const Tensor& weight,
+                         const Tensor& bias, bool training,
+                         double momentum, double eps) {
```

Comment:
I believe the functional `batch_norm` can take some default values:
```python
def batch_norm(input, running_mean, running_var, weight=None, bias=None,
               training=False, momentum=0.1, eps=1e-5):
```
it would be awesome to do the same in C++ as well :D (we can use `Tensor()` as null value for `const Tensor&` type)

Reply:
Thanks. I fixed it.